core/src/test/scala/better/files/DottyTests.scala

```diff
@@ -2,6 +2,32 @@ package better.files
 
 import scala.util.control.NonFatal
 
+case class Test(name: String, ignore: Boolean, result: () => Unit)
+
+abstract sealed class TestResult(test: Test) {
+  def color: String
+  def format: String = test.name
+  def print(): Unit = println(color + format + Console.RESET)
+}
+
+object TestResult {
+  case class Ignored(test: Test) extends TestResult(test) {
+    override def color = Console.YELLOW
+  }
+  case class Success(test: Test) extends TestResult(test) {
+    override def color = Console.GREEN
+
```

Comment:
You can blame silly @jvican https://github.com/olafurpg/scalafmt/issues/283

Disabling that shit by default soon.